### PR TITLE
property blacklist support

### PR DESCRIPTION
**Pull request by tdumitrescu**

### CODE REVIEW

File: src/mixpanel-core.js [hunk 1]
```diff
@@ -2424,6 +2424,10 @@ MixpanelLib.prototype.track = function(event_name, properties, callback) {
         , properties
     );
 
+    _.each(this.get_config('property_blacklist'), function(blacklisted_prop) {
```
Comment (jbwyme) [inline src/mixpanel-core.js]: I'd check to ensure that an array

**Comment by jbwyme** [comment]
Lgtm other than the defensive comment

File: src/mixpanel-core.js (hunk 1 above)
Comment (tdumitrescu) [inline src/mixpanel-core.js]: OK, but since it's given a default of `[]` it always should be

Comment (tdumitrescu) [inline src/mixpanel-core.js]: https://github.com/mixpanel/mixpanel-js/commit/96eab15bff256d9ac33e2a01a07f679d90e1adfd

**Pull request merged**